pygithub/im_commits_pygithub.py

```diff
@@ -0,0 +1,66 @@
+"""Demo designed to find and store information about PyGithub."""
+from github import Github
+from pydriller import RepositoryMining, GitRepository
+from pydriller.domain.commit import ModificationType
+from fastapi import FastAPI
+
+test_dict = {}
+def output_hash_map(dictionary):
+    """Method to print out the hash map."""
+    # print headings
+    print("Branch", "\t Author", "\t Time", "\t Files", "\t Message")
+    # prints hashmap content
+    # for key in dictionary:
+
+
+    for key, _ in dictionary.items():
+        print(key)
+        for items in dictionary[key]:
+            for item in items:
+                print("\t" + str(item) + "\t")
+            # print("\t" + str(item))
```

Comment:
This function is kind of hardcoded. I would suggest looking into pprint, which allows you to "pretty print" the `dictionary` here. Also, the function naming should be more intuitive here. Neither `output_hash_map` sound "pythonic", nor it actually reflects the purpose of this function, which is to print a dictionary in a formatted way right?